docs/modeling/design-time-code-generation-by-using-t4-text-templates.md

```diff
@@ -314,7 +314,7 @@ If you have installed the Visual Studio Modeling SDK, you can have all the templ
 [this blog post](https://devblogs.microsoft.com/devops/the-visual-studio-modeling-sdk-is-now-available-with-visual-studio-2017/).
 
 ```xml
-<Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v16.0\TextTemplating\Microsoft.TextTemplating.targets" />
+<Import Project="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v17.0\TextTemplating\Microsoft.TextTemplating.targets" />
```

Comment:
@Court72 Is there a way to make this conditional based on the value selected in the VS version selector for the docs? I.e. v17.0 for VS 2022 and v16.0 for VS 2019.

Reply:
Responded in Teams.